### PR TITLE
ERD-72: Configuration of proprietary jump targets

### PR DESCRIPTION
**Pull request by Waguramu**
**Goal**: erdblick needs to support jumping to proprietary coordinates, which are represented by closed-source formats.

**Scope**:
[ERD-72](https://youtrack.nds-association.org/issue/ERD-72) Configuration of proprietary jump targets
* Erdblick components and services 

**Deliverables**:
* Refactored app component.
* Menu items components.
* Information messages, map-related and jump targets Angular services.

**How to test**:
1. Build and run as usual (without a plugin).

OR

1. Copy plugin files from mapviewer repository to _config_ directory.
2. Build and run as usual (with a plugin).

OR

1. Test via mapviewer build (with a plugin).

**Notes**:
* _config_ directory needs to be present for correct path resolution by webpack (using dynamic import).

### CODE REVIEW

**Review by josephbirkner — APPROVED** [review]
Tested, looks great with some small nitpicks (see comments).

**Comment by josephbirkner** [comment]
> **Need to incorporate:** _unify canJump and jump and use something like toTargetCoords which returns an actual result it the target input is supported and null otherwise_

We can leave that out, the current solution is fine imo.

**Comment by josephbirkner** [comment]
We should also not forget to add a README.md section about config.json (also with respect to style sheets).

File: erdblick_app/app/menu.component.ts [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+        });
+
+        this.jumpToTargetService.availableOptions.subscribe((jumpTargets: Array<JumpTarget>) => {
+            this.searchItems = [
+                ...jumpTargets,
+                ...[
+                    {
+                        name: "Tile ID",
+                        label: "Jump to Tile by its Mapget ID",
+                        enabled: false,
```
[the remark this answers is not in the document]
Comment (Waguramu) [reply erdblick_app/app/menu.component.ts:45]: I'll consider turning JumpTarget interface into a class.

File: erdblick_app/app/menu.component.ts [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+            return [lat, lon, level];
+        }
+        return undefined;
+    }
+
+    jumpToWGS84(coordinates: number[] | undefined) {
+        if (coordinates === undefined) {
+            this.messageService.showError("Could not parse coordinates from the input.");
+            return;
+        } else {
```
[the remark this answers is not in the document]
Comment (Waguramu) [reply erdblick_app/app/menu.component.ts:177]: Will remove in the following PR.

**Pull request merged**